### ISSUE TITLE
TypeError: Cannot read property 'inputid' of undefined

### ISSUE THREAD

**Issue by jcristine** [haydendonald/blackmagic-atem-nodered#7]
Simply placed on atem node in flow, configured network at 192.168.0.7, named the network and atem nodes, clicked "deploy" and the node said "gathering information" and Node-Red crashed. 

Runtime:

Node-RED version: v1.0.6
Node.js  version: v12.18.0
Windows_NT 10.0.18363 x64 LE

Error:

7 Jun 15:39:38 - [info] [atem-network:dot7] Connected to ATEM @ 192.168.0.7
7 Jun 15:39:38 - [red] Uncaught Exception:
7 Jun 15:39:38 - TypeError: Cannot read property 'inputid' of undefined
    at Object.afterInit (C:\Users\User\.node-red\node_modules\blackmagic-atem-nodered\commands\multiViewerInput.js:109:106)
    at updateConnectionState (C:\Users\User\.node-red\node_modules\blackmagic-atem-nodered\network.js:127:58)
    at Socket.handleIncoming (C:\Users\User\.node-red\node_modules\blackmagic-atem-nodered\network.js:407:25)
    at Socket.emit (events.js:315:20)
    at UDP.onMessage [as onmessage] (dgram.js:910:8)

**Comment by haydendonald** [comment]
Hi,
What version are you running? The latest is version 2.2.6. I say this because i've already fixed this issue..

Having a look node red is displaying version 2.0.0 for some reason even though the latest version on npm is 2.2.6.  

I have requested node red to update it. For now type npm install blackmagic-atem-nodered and it should update to the latest version.

**Comment by haydendonald** [comment]
Ok node red has detected the version update. Not sure what happened there, try updating and see if the issue goes away :)

**Comment by jcristine** [comment]
I installed using "npm install blackmagic-atem-nodered" -- that works, thank you.  

The flow runs but I ran into another error.  Probably due to my misunderstanding of how to use.  I'm trying to run "marco 0" so I send this JSON to the atem node using an inject node:

{
    "payload": {
        "cmd": "macroAction",
        "data": {
            "macroId": 0,
            "action": "run"
        }
    }
}

When I initiate the "inject" I see the following on the node-red debug console:

10 Jun 09:33:36 - [info] [atem-network:dot7] Connected to ATEM @ 192.168.0.7
10 Jun 09:33:39 - [error] [atem-atem:Mini Pro] TypeError: Cannot read property 'toUpperCase' of undefined

![image](https://user-images.githubusercontent.com/12465410/84294751-79cb4080-aafe-11ea-90d7-0c8703f442c8.png)
